flixel/input/gamepad/FlxGamepadInputID.hx

```diff
@@ -56,14 +56,19 @@ abstract FlxGamepadInputID(Int) from Int to Int
 	var LEFT_TRIGGER         = 17;
 	var RIGHT_TRIGGER        = 18;
 	
+	/**identifier for the entire LEFT_ANALOG_STICK itself, not just any particular direction**/
 	var LEFT_ANALOG_STICK    = 19;
+	/**identifier for the entire RIGHT_ANALOG_STICK itself, not just any particular direction**/
 	var RIGHT_ANALOG_STICK   = 20;
 	
+	/**identifier for the entire DPAD itself, not just any particular button**/
+	var DPAD                 = 21;
```

Comment:
Yeah, there's some edge cases where it's useful. I'll follow up on Monday with some more about that.